modules/fluid_properties/src/fluidproperties/TabulatedFluidProperties.C

```diff
@@ -895,6 +918,100 @@ TabulatedFluidProperties::vaporPressure(Real temperature, Real & psat, Real & dp
     mooseError("vaporPressure not specified.");
 }
 
+Real
+TabulatedFluidProperties::vaporTemperature(Real pressure) const
+{
+
+  const FPDualReal p = pressure;
+
+  return vaporTemperature_ad(p).value();
```

Comment:
use the same forward as the others instead?

Reply:
addressed in [a6a0e70](https://github.com/idaholab/moose/pull/27506/commits/a6a0e70fd320f10a2453e722ddc0c69452dc09be)